generate_ics.py

```diff
@@ -1,58 +1,32 @@
 ## Adds your timetable from `data.txt` to Google Calendar.
 from __future__ import print_function
 import os
-
 import json
 import datetime
-import sys
-
-# this script works only with Python 3
-if sys.version_info[0] != 3:
-    print("This script works only with Python 3")
-    sys.exit(1)
-
 import re
```

Comment:
Also removed this part for obvious reasons ~hopefully no one uses Python 2 anymore~